src/universal/styles/ui.js

```diff
@@ -0,0 +1,15 @@
+
+// ui.js
+
+import theme from 'universal/styles/theme';
+
+const ui = {
```

Comment:
I think having this set of constants is _probably_ a good idea.

It might be a good idea to make a convention where any new color introduced here is imported from `theme`, rather than being declared here.